contracts.md

```diff
@@ -12,20 +12,29 @@ liquidity for OHM on Etherscan.
 
 ## sOHM
 
-https://etherscan.io/address/0x31932E6e45012476ba3A3A4953cbA62AeE77Fbbe
+https://etherscan.io/address/0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f
 
 You receive sOHM when you stake OHM at a 1:1 ratio. Adding this address to your
 wallet allows you to track your sOHM balance which increases with every rebase.
+Below are listed old sOHM contracts which got phased out due to historical
+reasons.
+
+- [0x3193...Fbbe](https://etherscan.io/address/0x31932E6e45012476ba3A3A4953cbA62AeE77Fbbe)
 
 ## aOHM
 
 https://etherscan.io/address/0x24ecfd535675f36ba1ab9c5d39b50dc097b0792e
 
-When OlympusDAO first launched, alphaOHM (aOHM) was used as a pre-allocation token which allowed the early participants to lay claim to OHM. Moving forward aOHM will serve as the in-game currency of [Alpha Omega](https://medium.com/@alpha_omega/alpha-omega-a-tale-of-two-cities-80a94966376b), a community-led social game that runs on the blockchain. Other than that aOHM is not relevant to OHM or the operation of OlympusDAO.
+When OlympusDAO first launched, alphaOHM (aOHM) was used as a pre-allocation
+token which allowed the early participants to lay claim to OHM. Moving forward
+aOHM will serve as the in-game currency of [Alpha Omega](https://medium.com/@alpha_omega/alpha-omega-a-tale-of-two-cities-80a94966376b),
+a community-led social game that runs on the blockchain. Other than that aOHM is
+not relevant to OHM or the operation of OlympusDAO.
 
 ## Staking
 
-https://etherscan.io/address/0x0822F3C03dcc24d200AFF33493Dc08d0e1f274A2
+new https://etherscan.io/address/0xFd31c7d00Ca47653c6Ce64Af53c1571f9C36566a
+old https://etherscan.io/address/0x0822F3C03dcc24d200AFF33493Dc08d0e1f274A2
```

Comment:
Only tracking the addresses now. Need to iterate here in separate PRs.

Reply:
judging from etherscan tx history, I think the former is used for staking tx only, while the latter unstaking

Reply:
I feel like there is some migration logic in there. Jeff didn't reply yet but we have to work on this section later anyway. Right now there is a staking helper involved though I don't know why that is and if it is only temporary.  I would revisit it later and only make sure we track the relevant addresses. Is there more action required for this here?

Reply:
go ahead then. We could add the staking helper later if needed. One more thing, I notice there is no newline between the old and new contracts. Is this intended?

![newline](https://user-images.githubusercontent.com/10302165/122241389-1506ab00-cef5-11eb-863b-d9c75b1e4cc4.png)

Reply:
Thank you for the feedback. Made it a list meanwhile. Also added the helper.